src/full/Agda/TypeChecking/Pretty/Warning.hs

```diff
@@ -108,11 +108,13 @@ prettyWarning = \case
       [prettyTCM d] ++ pwords "is not strictly positive, because it occurs"
       ++ [prettyTCM ocs]
 
-    NoEquivWhenSplitting doc -> vcat
-            [ fwords $ "Could not generate equivalence when splitting on indexed family, " ++
-                       "the function will not compute on transports by a path."
-            , nest 2 $ "Reason:" <+> pure doc
-            ]
+    UnsupportedIndexedMatch doc -> vcat
+      [ fsep (pwords "This clause uses pattern-matching features that are not yet supported by Cubical Agda,"
+           ++ pwords "the function to which it belongs will not compute when applied to transports."
+             )
+      , "\n"
```

Comment:
It is rather uncommon to use a newline character in a `Doc`.  Wouldn't the empty string `""` here have the same effect, to insert an blank line?  I'd think this is how `vcat` would treat it.

Reply:
I think I'm running into this rule:

> Above, except that if the last line of the first argument stops at least one position before the first line of the second begins, these two lines are overlapped.

So without the `\n`, it merged `Reason: ...` into the empty line. I fiddled with the formatting a bit to avoid that (and to align the justification properly)